db/migrate/20160327163400_create_feedback_files.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFeedbackFiles < ActiveRecord::Migration
+  def change
+    create_table :feedback_files do |t|
+      t.string :filename
+      t.binary :file_content
```

Comment:
This is good, but I think we'll need to store the content type as well now. https://gist.github.com/macek/610596